### ISSUE TITLE
Empty logs

### ISSUE THREAD

**Issue by Alexandrsv** [jesseduffield/lazydocker#574]
**Describe the bug**

**To Reproduce**
1. open lazydocker
2. Click on container
3. Log is empty

**Screenshots**
<details><summary>Sceen</summary>
<p>

![image](https://github.com/user-attachments/assets/dbeb7014-153f-48e8-88f8-9fc36660da4b)

</p>
</details> 

**Desktop (please complete the following information):**
 - OS: Ubuntu 24.04 LTS
 - Lazydocker Version 0.23.3

**Additional context**
Docker version 27.1.1, build 6312585
Containers are started using the command "docker compose up -d"

**Comment by Brenex** [comment]
Also experiencing this issue. Using the run command line as an alias under debian:
```
alias lzd='docker run --rm -it --name lazydocker -v /var/run/docker.sock:/var/run/docker.sock -v ~/docker-apps/lazydocker:/.config/jesseduffield/lazydocker lazyteam/lazydocker'
```

**Comment by slipperybeluga** [comment]
Common thread here seems to be using docker-compose -f. 

Lazydocker was working fine for me when I previously ran docker-compose up -d in my directory with a single docker-compose.yml file. Logs stopped showing up in lazydocker as I started using additional separate docker-compose files all started with -f {path} arguments. 

e.g., docker-compose $(for file in /path/*.yml; do echo -n "-f $file "; done) up -d

**Comment by zouloux** [comment]
Same here, containers started with `docker compose -f` have empty logs. Logs are visible with `docker logs $id --follow`. Debian 12, lazydocker installed with curl. Version 12.3. It was working in the previous version I had but I lost the specific version number.

**Comment by zouloux** [comment]
Tested with 12.1, same issue.

**Comment by RaymondBakker** [comment]
See https://github.com/jesseduffield/lazydocker/issues/391#issuecomment-1324799180

**Comment by Xerxes004** [comment]
> See [#391 (comment)](https://github.com/jesseduffield/lazydocker/issues/391#issuecomment-1324799180)

I don't think this is the same issue. I just started the service with `docker compose up -d` and saw no logs.

**Comment by sequencerr** [comment]
I don't see logs with bash alias approach, but it works for manually installed binary
